functions/notify_slack.py

```diff
@@ -51,6 +51,8 @@ def ecs_notification(message, region):
     for key in message['detail']['containers']:
       if 'reason' in key:
         fields.append( { "title": "reason", "value": key['reason'], "short": True } )
+        fields.append( { "title": "taskArn", "value": key['taskArn'], "short": True } )
+        fields.append( { "title": "containerArn", "value": key['containerArn'], "short": True } )
```

Comment:
I wish there was a more readable identifier like `ContainerName`

Reply:
Hmmm there might be. It's referenced as just `name` within the json. I'll look into that.

Reply:
Yea within the containers mapping (?) container name is referenced as just name which we can replace here. 👍